src/cpp/transport/tcp/TCPServerWindows.cpp

```diff
@@ -29,10 +29,14 @@ TCPServer::TCPServer(uint16_t port)
       listener_thread_(),
       running_cond_(false),
       messages_queue_{}
-{}
+{
+    (void) discovery_port;
+}
 
-bool TCPServer::init()
+bool TCPServer::init(bool discovery_enabled)
 {
+    (void) discovery_enabled;
```

Comment:
Should we avoid windows servers if the call was used with discovery_enabled? Returning an error maybe.

Reply:
This is a temporal workaround until the agent support discovery for windows. The documentation has been changed to specify that currently is only supported on linux: https://github.com/eProsima/Micro-XRCE-DDS-docs/pull/23